core/utils.js

```diff
@@ -788,7 +813,7 @@ Blockly.utils.wrapToText_ = function(words, wordBreaks) {
  * @param {string} text The actual text to measure
  * @return {number} Width of the text in px.
  */
-Blockly.measureText = function(fontSize, fontFamily, fontWeight, text) {
+Blockly.utils.measureText = function(fontSize, fontFamily, fontWeight, text) {
```

Comment:
Because this and `encodeEntities` are used in Scratch Blocks but do not exist in Blockly I recommend that we add notes in the comments to help with future merges.